readme.md

```diff
@@ -12,33 +12,36 @@
 ## Authors
 
 - [@TrivCodez](https://github.com/TrivCodez/)
+
 ## Contributing
 
 Contributions are always welcome!
 
 See `contributing.md` for ways to get started.
+
 ## FAQ
 
 #### What is this repo?
 
-This repo is a repo containing ways you can use chatgpt for anything! We will provide high-quality prompts for each category such as development, art ideas, etc. 
+This repo is a repo containing ways you can use chatgpt for anything! We will provide high-quality prompts for each category such as development, art ideas, etc.
```

Comment:
```
This repo contains high-quality prompts you can use for ChatGPT!
We also provide multiple different prompts for each category such as code generation, development, art ideas, etc.
```

i'd personally recommend capitalising where needed and i've also gone ahead and simplified things a little bit.